zipline/finance/slippage.py

```diff
@@ -502,3 +502,30 @@ def get_simulated_impact(self,
     def get_txn_volume(self, data, order):
         volume = data.current(order.asset, 'volume')
         return volume * self.volume_limit
+
+
+class FixedBasisPointsSlippage(SlippageModel):
+    """
+    Model slippage as a fixed percentage of fill price.
+
+    Parameters
+    ----------
+    bps : int, optional
+        basis points to apply
+    """
+    def __init__(self, bps=5):
+        super(FixedBasisPointsSlippage, self).__init__()
+        self.bps = bps
+
+    def __repr__(self):
+        return '{class_name}(basis_points={bps})'.format(
+            class_name=self.__class__.__name__, spread=self.bps,
+        )
+
+    def process_order(self, data, order):
+        price = data.current(order.asset, "close")
+
+        return (
+            price + price * (self.bps / 10000.0 * order.direction),
```

Comment:
Small perf note: there's no reason to do this division on every order we process; we can just do this once at construction time.

Reply:
I'm wondering if we should be capping the percentage of the bar that we fill here.  I know that part of the goal of adding this model is to give a reasonable estimate without simulating a huge number of transactions, but it seems pretty inaccurate to fill a 1000 share order in a minute where only 10 shares transacted.

Reply:
- added `self.percentage` to avoid doing division on every order
- will change the slippage model depending on what we decide about that